tests/unit/requests/test__helpers.py

```diff
@@ -55,48 +55,6 @@ def test__get_body_w_content_consumed(self):
         assert body == _request_helpers.RawRequestsMixin._get_body(response)
 
 
-def test_http_request():
-    transport, responses = _make_transport(http.client.OK)
-    method = "POST"
-    url = "http://test.invalid"
-    data = mock.sentinel.data
-    headers = {"one": "fish", "blue": "fish"}
-    timeout = mock.sentinel.timeout
-    ret_val = _request_helpers.http_request(
-        transport,
-        method,
-        url,
-        data=data,
-        headers=headers,
-        extra1=b"work",
-        extra2=125.5,
-        timeout=timeout,
-    )
-
-    assert ret_val is responses[0]
-    transport.request.assert_called_once_with(
-        method,
-        url,
-        data=data,
-        headers=headers,
-        extra1=b"work",
-        extra2=125.5,
-        timeout=timeout,
-    )
-
-
-def test_http_request_defaults():
-    transport, responses = _make_transport(http.client.OK)
-    method = "POST"
-    url = "http://test.invalid"
-    ret_val = _request_helpers.http_request(transport, method, url)
-
-    assert ret_val is responses[0]
-    transport.request.assert_called_once_with(
-        method, url, data=None, headers=None, timeout=EXPECTED_TIMEOUT
-    )
-
-
```

Comment:
`_make_transport` (defined at line 62 below) no longer has any callers, and should be removed.